plugins/ocp/ocp-nvme.c

```diff
@@ -0,0 +1,672 @@
+#include <stdio.h>
```

Comment:
Could you add the SPDIX license identifier to this file and the header file as well?